### PR TITLE
Raster-Vision Pipeline for Training STACs

### LINKED ISSUE #4: Train RasterVision model on Sen1Floods11
This issue is to train a flood inundation model on the output of https://github.com/azavea/noaa-flood-mapping/issues/2

This model will be the baseline benchmark for the project.
Additionally it will be the first time we source Sentinel 1 VV and VH bands as features and will help flush out any potential issues.

- [ ] Train Sen1Flood11 model from provided data.
- [ ] Use training, test, validation split in STAC format.
- [ ] Produce segmentation STAC as output (so it may be indexed with Franklin)

### PR DESCRIPTION
**Pull request by jamesmcclain**
```bash
docker run -it --rm -v $HOME/.aws:/root/.aws:ro  -v $(pwd):/workdir -w /workdir raster-vision:pytorch-pystac-45815af bash
rastervision run local /workdir/rv_stac_config.py -a root_uri /tmp/xxx -a catalog_root /vsitar//vsis3/.../catalogs.tar/mldata_hand/catalog.json
```

Depends on #66 
Closes https://github.com/azavea/noaa-flood-mapping/issues/4

### CODE REVIEW

File: inference/README.md [hunk 1]
```diff
@@ -15,17 +15,11 @@ Start the container ensuring:
...
 
 ```bash
-docker run --rm -it \
-  -e OUTPUT_ROOT_URI=/opt/data/output \
-  -e CATALOG_ROOT_URI=/opt/src/code/ml_stac/catalog.json \
-  -v ~/.aws:/root/.aws \
-  -v ${RV_CODE_DIR}:/opt/src/code \
-  -v ${RV_OUT_DIR}:/opt/data/output \
-  raster-vision:pytorch-pystac-v0.4.0 /bin/bash
+docker run -it --rm -v $HOME/.aws:/root/.aws:ro -w /workdir raster-vision:pytorch-pystac-45815af bash
```
Comment (moradology) [inline inference/README.md:18]: Less is more. Good choice here

File: inference/rv_stac_config.py [hunk 2]
```diff
@@ -93,62 +97,60 @@ def build_dataset_from_catalog(
...
-    if catalog_root is None:
-        sys.exit("No catalog URI set. Set the environment variable CATALOG_ROOT_URI")
-
+def get_config(runner, root_uri, catalog_root, hours):
     # Read STAC catalog
     catalog: Catalog = Catalog.from_file(catalog_root)
 
     # TODO: pull desired channels from root collection properties
-    channel_ordering: [int] = [0, 1, 2]
+    channel_ordering: [int] = [0, 1, 1]
```
Comment (moradology) [inline inference/rv_stac_config.py:127]: Is this duplication intentional?

**Review by moradology — APPROVED** [review]
One small question; good to merge after verification of that

File: inference/rv_stac_config.py (hunk 2 above)
Comment (jamesmcclain) [reply inference/rv_stac_config.py:127]: Yeah, I discovered that the multiband functionality does not kick-in until the number of channels is > 3.  The totally pretrained models expect 3 channels, so one is forced to duplicate.

**Pull request merged**